Week2/homework/map_filter.js

```diff
@@ -2,4 +2,10 @@
 
 const numbers = [1, 2, 3, 4];
 
-// Add your code here
+const newNumbers = numbers.filter(function (number) {
```

Comment:
It is working well.

Instead of doing it this way you could have done it this way:

const newNumbers = numbers.filter(number => number % 2 !== 0).map(number => number * 2);

Try to review and understand it ;)